cmd/kubectl-convert/kubectl-convert.go

```diff
@@ -17,34 +17,29 @@ limitations under the License.
 package main
 
 import (
-	goflag "flag"
 	"os"
 
 	"github.com/spf13/pflag"
 
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	cliflag "k8s.io/component-base/cli/flag"
+	"k8s.io/component-base/logs"
 	cmdutil "k8s.io/kubectl/pkg/cmd/util"
-	"k8s.io/kubectl/pkg/util/logs"
 	"k8s.io/kubernetes/pkg/kubectl/cmd/convert"
 )
 
 func main() {
 	flags := pflag.NewFlagSet("kubectl-convert", pflag.ExitOnError)
+	flags.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
 	pflag.CommandLine = flags
 
 	kubeConfigFlags := genericclioptions.NewConfigFlags(true).WithDeprecatedPasswordFlag()
 	kubeConfigFlags.AddFlags(flags)
+	logs.AddFlags(flags)
```

Comment:
Note that this adds `--log-flush-frequency` to the command line flags. Previously, that flag got lost when overwriting `pflag.CommandLine`.

However, the flag is not working properly yet because its value gets used in `logs.InitLogs` *before* parsing command line flags. This may be a common problem. Needs more thought on how to avoid this.

Reply:
I fixed the problem with `--log-flush-frequency` not having an effect in https://github.com/kubernetes/kubernetes/pull/105076 and in this PR avoided adding it to kubectl-convert.